modules/loki-rbac-permissions.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/custom-logging-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="loki-rbac-permissions_{context}"]
+= Authorizing Loki rules RBAC permissions
+
+In logging 5.7 and later, the Cluster Logging Operator provides `alertingrule-editor-role` and `recordingrule-editor-role` cluster roles, which enable users to modify alerting and recording rules for the LokiStack.
```

Comment:
I think we found out this week, that these roles do not actually exist. I had seen them in "Loki Operator" and assumed that they would be deployed using OLM, but this does not seem to be the case and never was.

@JoaoBraveCoding You did verify this yesterday, right?

I think, for now, the easiest solution would be to remove the paragraph about RBAC rules referencing the `alertingrule-editor-role` and `recordingrule-editor-role` until we have something better (maybe when the 5.8 release comes around).

Reply:
Yup, I forgot to send an update here! Thanks for picking up my slack

Reply:
Commented out the paragraph - we can revisit post 5.8 if you wouldn't mind opening an issue for it in Jira.